include/flatbuffers/flatbuffers.h

```diff
@@ -1265,7 +1271,7 @@ class Verifier FLATBUFFERS_FINAL_CLASS {
     // Call T::Verify, which must be in the generated code for this type.
     return Verify<uoffset_t>(buf_) &&
       reinterpret_cast<const T *>(buf_ + ReadScalar<uoffset_t>(buf_))->
-        Verify(*this);
+        Verify(*this) && GetSize();
```

Comment:
They test for different things, and as I said, this size computation should be optional.